source/docs/casper/developers/prerequisites.md

```diff
@@ -11,64 +13,307 @@ This section covers:
 3. Setting up an Account on a Casper network
 4. Acquiring the IP address of a peer on the official Testnet or Mainnet  
 
-## Installing Rust {#installing-rust}
+To be able to develop comfortably on the Casper Network you should use either `macOS` or `Linux Ubuntu 20.04`. Developing on Windows is not advised.
 
-On the Casper platform, developers may write smart contracts in any language that compiles to Wasm. These How To guides focus on code examples that use Rust and a Rust client to interact with a Casper network. While following these guides, we recommend setting up Rust and installing all dependencies. For step-by-step instructions, visit [Getting Started with Rust](./writing-onchain-code/getting-started.md).
+:::caution
 
-## Casper Command-line Client {#the-casper-command-line-client}
+Casper Network does not officially support `macOS`. If you encounter any problems reach out to the community on Telegram or Discord.
 
-You can find the default Casper client on [crates.io](https://crates.io/crates/casper-client). This client communicates with the network to transmit your deploys.
+:::
 
-Run the commands below to install the Casper client on most flavors of Linux and macOS. You should have [Rust](https://www.rust-lang.org/tools/install) installed, otherwise check the [alternative installation methods](#alternative-installation) below.
+Follow the steps below to install necessary software for the development environment.
+
+## Install the environment on macOS / Linux
+
+## Step 1.
+
+### Install xcode command line tools on macOS {#install-xcode}
+
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+xcode-select --install
+```
+
+</TabItem>
+</Tabs>
+
+Check the version 
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+xcode-select -p
+```
+
+</TabItem>
+</Tabs>
+
+### Get curl on Linux {#install-curl}
+
+<Tabs>
+<TabItem value="Linux" label="Linux">
+
+```bash
+sudo apt install curl
+```
+</TabItem>
+</Tabs>
+
+## Step 2.
+
+### Install rust {#install-rust}
+
+<Tabs>
+<TabItem value="macOS" label="macOS">
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+</TabItem>
+<TabItem value="Linux" label="Linux">
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+</TabItem>
+</Tabs>
+
```

Comment:
In the tutorial improved by @jonas089 we also have:

```suggestion
The installation script automatically adds Rust to your system PATH after your next login.
To start using Rust right away instead of restarting your terminal, run the following command in your shell to add Rust to your system PATH manually:
```
But I leave it up to you.

Reply:
Is this correct? Do we have to logout and login to use Rust' Or is it simply a restart of the terminal/bash shell that is required? I have a feeling it is the latter...

Reply:
The suggested comments have been added.